app/models/category.rb

```diff
@@ -5,6 +5,7 @@ class Category < ApplicationRecord
                       dependent: :destroy
 
   scope :parents, ->{where("parent_id is null")}
+  scope :childrens, ->{where("parent_id is not null")}
```

Comment:
`{where("parent_id is not null")}` -> `>{where.not parent_id: nil)}` 
Nên dùng sự hỗ trợ từ rails nhé. Vì nếu có change DB thì ko ảnh hưởng tới code của scope.